tasks/ffmpeg.py

```diff
@@ -46,6 +46,7 @@ def ffmpeg(ctx, clean=False):
         "--disable-stripping",
         "--disable-programs",
         "--disable-doc",
+        "--disable-zlib",
```

Comment:
`zlib` in FFmpeg's configure is set to `"autodetect"`. Now that we actually build a version of `zlib`, FFmpeg function's compilation was missing `zlib`'s symbols. Rather than link with our `zlib`, we explicitly disabled.